packages/DatePicker/src/components/Calendar/Calendar.styles.js

```diff
@@ -1,17 +1,57 @@
 import styled from "styled-components";
 import tokens from "@paprika/tokens";
 import stylers from "@paprika/stylers";
+import Button from "@paprika/button";
 
 import CalendarBaseStyle from "./CalendarBase.styles";
+import { HoveredItemStyles, SelectedItemStyles } from "../../shared.styles";
+
+export const ArrowIconStyles = `
+  > svg {
+    display: block;
+  }
+`;
+
+const IconButtonStyles = `
+  transition: background-color 0.2s ease-out;
+  
+
+  &:hover {
+    background-color: ${stylers.alpha(tokens.color.black, 0.1)}; 
+    border-color: ${tokens.border.hoverColor};
+  }
+
+  &:focus,
+  &:active {
+    box-shadow: ${tokens.highlight.active.withBorder.boxShadow};
+    outline: none;
+  }
+
+  &:active {
+    background-color: ${stylers.alpha(tokens.color.black, 0.2)};
+  }
+`;
 
 const CalendarStyled = styled.div`
   ${CalendarBaseStyle}
 
+  ${({ shouldHidden }) =>
+    shouldHidden &&
+    `
+    width: 0;
+    height: 0;
+    overflow: hidden;
+  `}
+
   * {
     box-sizing: border-box;
     font-family: ${tokens.fontFamily.default};
```

Comment:
Incidental again... but would `inherit` work here instead, so this component remains agnostic of the app's typeface?

Reply:
I moved ` font-family: ${tokens.fontFamily.default};` to `DatePicker.styles.js`. I think it's good to set the default `font-family` otherwise I need to set it in the date picker example in storybook.

Reply:
Does the component _need_ to be a specific font? Ideally when we switch our apps to use `IBM Plex`, this component shouldn't have to be changed.  In fact we could use it in different apps with different UI fonts, and the component should adapt.  Ideally.

Reply:
🤔Sure! The only thing is some font doesn't works well with `bold` on hover, for example `Helvetica Neue`. It will shift a little bit as you said before. `IBM Plex` and `Lato` doesn't have this issue. I will just use inherit then. I can add the default font in storybook if that looks bad.